iXBRLViewerPlugin/iXBRLViewer.py

```diff
@@ -219,9 +222,21 @@ def createViewer(self, scriptUrl="js/dist/ixbrlviewer.js"):
             }
 
             factData = {
-                "v": f.value if not f.isNil else None,
                 "a": aspects,
             }
+
+            if f.isNil:
+                factData["v"] = None
+            elif f.concept is not None and f.concept.isEnumeration:
+                qnEnums = f.xValue
+                if not isinstance(qnEnums, list):
+                    qnEnums = (qnEnums,)
```

Comment:
What does this do if the fact value is invalid?  Do we need some kind of `qnEnums is not None`?